sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ClasspathUtil.java

```diff
@@ -9,11 +9,18 @@
 
 final class ClasspathUtil {
 
-  @SuppressWarnings("UnusedException")
-  static void checkClassExists(String className, String featureName, String requiredLibrary) {
+  static boolean checkClassExists(String className) {
```

Comment:
Nit: both of these methods are named the same while being completely different in what they do - one of them does not throw and you have to check the return value, the other one does not return anything and the caller has to rely on an exception being thrown.
How about renaming this one to `classExists` or `doesClassExist`?

Reply:
Good point. I like `classExists`. The `check` prefix is associated with methods that throw if some condition is not met, i.e. `Preconditions.checkArgument`.

Reply:
Turns out that there's just a single use of `ClasspathUtil` left and its a call to `ClasspathUtil.classExists`. I've moved the `try / catch` logic to the calling location and deleted `ClaspathUtil`.